### PR TITLE
feat: Display unmanaged experiments with label

### PR DESCRIPTION
**Pull request by gt2345**
## Description

Add label to unmanaged experiments in new experiment list table and experiment details page.

Screenshot 2023-09-10 at 12 40 11 PM

Screenshot 2023-09-07 at 6 48 54 PM

Screenshot 2023-09-11 at 5 20 33 PM

## Test Plan

Unmanaged experiment should have labels.

## Commentary (optional)

Since experiment list table is canvas, it would require some extra investigation to add a tooltip there.

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Review by erikwilson — APPROVED** [review]

File: webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx [hunk 1]
```diff
@@ -517,6 +518,11 @@ const ExperimentDetailsHeader: React.FC<Props> = ({
             <div className={css.name} role="experimentName">
               {experiment.name}
             </div>
+            {experiment.unmanaged && (
+              <Badge tooltip="Workload not managed by Determined" type="Header">
```
Comment (keita-determined) [inline webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx:522]: Suggestion: different color is more visible in dark mode

image

File: webui/react/src/pages/F_ExpList/glide-table/custom-renderers/cells/linkCell.tsx [hunk 2]
```diff
@@ -87,6 +93,23 @@ const renderer: CustomRenderer<LinkCell> = {
...
+      ctx.beginPath();
+      roundedRect(ctx, x, y, measureTextCached(TAG_CONTENT, ctx).width + 8, TAG_HEIGHT, 4);
+      ctx.stroke();
+      ctx.fill();
+      ctx.fillStyle = '#fff';
+      ctx.fillText(
+        TAG_CONTENT,
+        x + 4,
+        y + TAG_HEIGHT / 2 + getMiddleCenterBias(ctx, `12px ${theme.fontFamily}`),
+      );
```
Comment (keita-determined) [inline webui/react/src/pages/F_ExpList/glide-table/custom-renderers/cells/linkCell.tsx:111]: dont we need `ctx. closePath();`?

File: webui/react/src/pages/F_ExpList/glide-table/custom-renderers/cells/linkCell.tsx [hunk 3]
```diff
@@ -87,6 +93,29 @@ const renderer: CustomRenderer<LinkCell> = {
...
     ctx.fillStyle = theme.linkColor;
     ctx.fillText(link.title, drawX, drawY);
+    if (link.unmanaged) {
+      const x = drawX + commaMetrics.width + 8;
+      const y = drawY - TAG_HEIGHT / 2;
+      ctx.fillStyle = '#132231';
+      ctx.lineWidth = 2;
+      ctx.strokeStyle = theme.textBubble;
+      ctx.beginPath();
+      const tagFont = `600 11px ${theme.fontFamily}`;
```
Comment (hkang1) [inline webui/react/src/pages/F_ExpList/glide-table/custom-renderers/cells/linkCell.tsx:103]: **nit suggestion:** in javascript, there are a lot of hoisting going on (where variable and function declarations are moved to the top) so in code it's common practice to have all the declarations like `const x = y` etc all grouped together at the top of the block whenever possible. There are exceptions such as when the declaration has a dependency on another runtime code, needing it to follow after that execution.

**Review by hkang1 — COMMENTED** [review]
LGTM in addition to what Keita suggested, one minor nit

File: webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx (hunk 1 above)
Comment (gt2345) [reply webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx:522]: Screenshot 2023-09-11 at 4 29 36 PM
Border added as suggested by Michiel

File: webui/react/src/pages/F_ExpList/glide-table/custom-renderers/cells/linkCell.tsx (hunk 3 above)
Comment (gt2345) [reply webui/react/src/pages/F_ExpList/glide-table/custom-renderers/cells/linkCell.tsx:103]: Cool, that's great suggestion, I grouped them together under `if (link.unmanaged)` to make it obvious that some values are only related to unmanaged experiments. 
Regarding not able to view the unmanaged tag for experiment list table, have you rebuilt the backend from this branch?

**Review by keita-determined — APPROVED** [review]
LGTM
Could you link a jira ticket in the description?

**Pull request merged**